shared/featureconfig/flags.go

```diff
@@ -309,9 +309,17 @@ var (
 		Usage:  deprecatedUsage,
 		Hidden: true,
 	}
+	// UseSpanCacheFlag enables the slasher to use span cache.
+	deprecatedUseSpanCacheFlag = &cli.BoolFlag{
+		Name:   "span-map-cache",
+		Usage:  deprecatedUsage,
+		Hidden: true,
+	}
 )
 
-var deprecatedFlags = []cli.Flag{
+// DeprecatedFlags list of old flags that are not being used in codebase
+// anymore.
+var DeprecatedFlags = []cli.Flag{
```

Comment:
lets not export this, can you follow the same pattern for validator/beacon-node

Reply:
removed

Reply:
fixed